uimaj-core/src/main/java/org/apache/uima/resource/impl/PearSpecifier_impl.java

```diff
@@ -38,38 +38,36 @@
   private String mPearPath;
 
   private Parameter[] mParameters;
+  private NameValuePair[] mPearParameters;
 
   /**
    * Creates a new <code>PearSpecifier_impl</code>.
    */
   public PearSpecifier_impl() {
   }
 
-  /**
-   * @return Returns the Parameters.
-   */
```

Comment:
What about leaving the javadoc comment but adding a Javadoc @deprecated element which specifies what do instead?

Reply:
The deprecation info including JavaDoc is inherited from the interface.